ceos_ard_cli/utils/files.py

```diff
@@ -22,7 +22,7 @@ def write_file(file, content):
 
 def get_all_folders(folder, deep=True):
     folders = []
-    for f in Path(folder).iterdir():
+    for f in folder.iterdir():
```

Comment:
```suggestion
    for f in Path(folder).iterdir():
```

We should ensure that the input is actually a path. If we input a string, this line would fail.